docs/data/material/getting-started/templates/dashboard/getDashboardTheme.tsx

```diff
@@ -0,0 +1,701 @@
+    background: {
+      default: 'hsl(0, 0%, 100%)',
+      paper: gray[50],
+      ...(mode === 'dark' && { default: 'hsl(220, 30%, 3%)', paper: gray[900] }),
+    },
+    text: {
+      primary: gray[800],
+      secondary: gray[600],
+      ...(mode === 'dark' && { primary: 'hsl(0, 0%, 100%)', secondary: gray[400] }),
+    },
+    action: {
+      selected: `${alpha(brand[200], 0.2)}`,
+      ...(mode === 'dark' && {
+        selected: alpha(brand[800], 0.2),
+      }),
+    },
+  },
+  typography: {
+    fontFamily: ['"Inter", "sans-serif"'].join(','),
+    h1: {
+      fontSize: customTheme.typography.pxToRem(60),
+      fontWeight: 600,
+      lineHeight: 1.2,
+      letterSpacing: -0.5,
+    },
+    h2: {
+      fontSize: customTheme.typography.pxToRem(48),
+      fontWeight: 600,
+      lineHeight: 1.2,
+    },
+    h3: {
+      fontSize: customTheme.typography.pxToRem(42),
+      lineHeight: 1.2,
+    },
+    h4: {
+      fontSize: customTheme.typography.pxToRem(36),
+      fontWeight: 500,
+      lineHeight: 1.5,
+    },
+    h5: {
+      fontSize: customTheme.typography.pxToRem(20),
+      fontWeight: 600,
+    },
+    h6: {
+      fontSize: customTheme.typography.pxToRem(18),
+    },
+    subtitle1: {
+      fontSize: customTheme.typography.pxToRem(18),
+    },
+    subtitle2: {
+      fontSize: customTheme.typography.pxToRem(16),
+    },
+    body1: {
+      fontSize: customTheme.typography.pxToRem(15),
+      fontWeight: 400,
+    },
+    body2: {
+      fontSize: customTheme.typography.pxToRem(14),
+      fontWeight: 400,
+    },
+    caption: {
+      fontSize: customTheme.typography.pxToRem(12),
+      fontWeight: 400,
+    },
+  },
+  shape: {
+    borderRadius: 12,
+  },
+});
+
+export default function getDashboardTheme(mode: PaletteMode): ThemeOptions {
+  return {
+    ...getDesignTokens(mode),
+    components: {
+      MuiButtonBase: {
+        defaultProps: {
+          disableTouchRipple: true,
+          disableRipple: true,
+        },
+        styleOverrides: {
+          root: {
+            boxSizing: 'border-box',
+            transition: 'all 100ms ease-in',
+            '&:focus-visible': {
+              outline: `3px solid ${alpha(brand[500], 0.5)}`,
+              outlineOffset: '2px',
+            },
+          },
+        },
+      },
+      MuiButton: {
+        styleOverrides: {
+          root: ({ theme, ownerState }) => ({
+            boxSizing: 'border-box',
+            boxShadow: 'none',
+            borderRadius: '10px',
+            textTransform: 'none',
+            '&:active': {
+              transform: 'scale(0.98)',
+            },
+            ...(ownerState.size === 'small' && {
+              maxHeight: '32px',
+            }),
+            ...(ownerState.size === 'medium' && {
+              height: '40px',
+            }),
+            ...(ownerState.variant === 'contained' &&
+              ownerState.color === 'primary' && {
```

Comment:
We should probably sort out the inconsitencies between the outlined and contained buttons 🙈 The outlined version is currently grey, following the design, but I didn't touch the contained primary and it's using the brand colors.

Reply:
I think I'll create a page for this, like[ Danilo did with the branding theme](https://master--material-ui.netlify.app/experiments/website/branding-theme-test/), so we can have a better view of the components' consistency.

Reply:
[Done!](https://deploy-preview-41757--material-ui.netlify.app/experiments/website/dashboard-template-theme/) ✅

Reply:
That's super helpful 🎉 I created a new variant for the cards and added it there 👍